docs/administrator/security/verify-artifacts.md

```diff
@@ -73,4 +73,83 @@ For more information, please refer to [k8s image pull policy](https://kubernetes
 The image verification process can also be implemented using the [sigstore policy-controller](https://docs.sigstore.dev/policy-controller/overview) controller during deployment. Here are some resources to help you get started with `policy-controller`:
 
 - [Install](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
-- [configuration options](https://github.com/sigstore/policy-controller/tree/main/config)
\ No newline at end of file
+- [configuration options](https://github.com/sigstore/policy-controller/tree/main/config)
+
+## SBOM
+
+SBOM, a Software Bill of Materials, is an inventory of all of the components that exist within a software resource, such as third-party libraries or modules, and has emerged as a key building block in software security and software supply chain risk management. 
+
+Starting with release `v1.10.2`, SBOM for Karmada projects will be available in Karmada's release Assets. Integrated with different tools, we can get the information of:
+
+- List of Components and Dependencies
+- Version Information
+- Licenses
+- Dependency Trees/Graphs
+
+and so on. Below are two examples of using tools to parse karmada's SBOM.
+
+### Prerequisites
+
+You need to install the following tools:
+
+- `bom` ([Installation Guide](https://github.com/kubernetes-sigs/bom#installation))
+- `trivy` ([Installation Guide](https://aquasecurity.github.io/trivy/v0.52/getting-started/installation/))
```

Comment:
```suggestion
- `bom` (see [Installation Guide](https://github.com/kubernetes-sigs/bom#installation))
- `trivy` (see [Installation Guide](https://aquasecurity.github.io/trivy/v0.52/getting-started/installation/))
```

Reply:
Would it be better to stay consistent with https://github.com/karmada-io/website/blob/839c421a05042bb9954d83e54242cbc2382dba9d/docs/administrator/security/verify-artifacts.md?plain=1#L13-L15